crates/evm/src/evm/system_contracts/src/Bridge.sol

```diff
@@ -171,29 +171,29 @@ contract Bridge is Ownable2StepUpgradeable {
     /// @param a First byte sequence
     /// @param b Second byte sequence
     function isBytesEqual(bytes memory a, bytes memory b) internal pure returns (bool result) {
-        require(a.length == b.length, "Lengths do not match");
+        uint256 len = a.length;
+        if (len != b.length) {
+            return false;
+        }
 
-        uint256 fullChunks = a.length / 32;
-        for (uint256 i = 0; i < fullChunks; i++) {
-            bytes32 chunkA;
-            bytes32 chunkB;
+        uint256 offset = 32;
```

Comment:
In case a,b len = 32, cmp will be 1 per byte?

Reply:
just made offset < len to offset <= len. I think that solves this